Lib/test/test_tempfile.py

```diff
@@ -932,8 +932,14 @@ def do_create(self, dir=None, pre="", suf="", delete=True):
                                            delete=delete)
 
         self.nameCheck(file.name, dir, pre, suf)
+        self.nameCheck(os.fspath(file), dir, pre, suf)
         return file
 
+    def test_pathlike(self):
+        tmp = self.do_create()
+        self.assertTrue(isinstance(tmp, os.PathLike))
+        cls = type(tmp)
+        self.assertTrue(issubclass(cls, os.PathLike))
```

Comment:
```suggestion
        self.assertIsInstance(tmp, os.PathLike)
```

Is it possible for the `isinstance()` check to pass but the `issubclass()` test to fail?